src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,222 @@
+import { Component, Prop, h } from '@stencil/core';
+import { InfoTileProgress } from './info-tile.types';
+import { Link } from '@limetech/lime-elements';
+
+/**
+ * This component can be used on places such as a start page or a dashboard.
+ * It offers features for visualizing aggregated data along with supplementary
+ * information.
+ *
+ * If clicking on the component should navigate the user to
+ * a new screen or web page, you need to provide a URL,
+ * using the `link` property.
+ *
+ * @exampleComponent limel-example-info-tile
+ * @exampleComponent limel-example-info-tile-badge
+ * @exampleComponent limel-example-info-tile-progress
+ * @exampleComponent limel-example-info-tile-loading
+ * @exampleComponent limel-example-info-tile-styling
+ */
+@Component({
+    tag: 'limel-info-tile',
+    shadow: true,
+    styleUrl: 'info-tile.scss',
+})
+export class InfoTile {
+    /**
+     * A piece of text or number that is the main piece of information
+     * which the component is intended to visualize.
+     */
+    @Prop({ reflect: true })
+    public value: number | string;
+
+    /**
+     * Name of icon for the info tile.
+     */
+    @Prop()
+    public icon?: string;
+
+    /**
+     * The text to show below the info tile. Long labels will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label?: string = null;
+
+    /**
+     * A string of text that is visually placed before the value.
+     */
+    @Prop({ reflect: true })
+    public prefix?: string;
+
+    /**
+     * A string of text that is visually placed after the value.
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * Set to `true` if info tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled? = false;
+
+    /**
+     * If supplied, the info tile will display a notification badge.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    /**
+     * Set to `true` to put the component in the `loading` state.
+     * This does _not_ disable the link. To do so, the
+     * `disabled` property should be set to `true` as well.
+     */
+    @Prop({ reflect: true })
+    public loading? = false;
+
+    /**
+     * If supplied, the info tile will be a clickable link.
+     *
+     * Supplying a value also adds an elevated effect using a shadow,
+     * as well as `cursor: pointer`, which appears on hover.
+     * While we strongly recommend supplying a link whenever the
+     * component should act as a link, if this is not possible, and
+     * you need to provide interaction through a click handler,
+     * you can still get the correct styling by supplying a `Link`
+     * object with the `href` property set to `'#'`.
+     */
+    @Prop()
```

Comment:
just curious. don't we need `@Prop({ reflect: true })` here? or we can't use it?

Reply:
You can only use `reflect` for primitives, i.e. `string`, `number` and `boolean`

Reply:
Found an email notification about this thread when I was trying to clean up my inbox a bit. Just wanted to clarify that you _can_ use `reflect` for complex values (complex values = all values that are not primitives), but you should avoid it, because it leads to lower performance because these complex properties need to be serialised to strings all the time.

Reply:
I don't think you can since Stencil does not know how to serialize them, they will just en up as the string `[Object object]`

Reply:
Ah, ok, that may well be true. I don't think I've actually looked at what happens, I only know the compiler doesn't balk if you try to do it 😄